File: CloudFlow/FrontEnd-App/src/app/shared/model/reports/nivel-reprezentare-comisii-report.model.ts
import { JsonDateConverter } from "@app/shared/json-mapper";
import { JsonObject, JsonProperty } from "json2typescript";
import { NivelReprezentareComisiiReportRowModel } from "./nivel-reprezentare-comisii-report-row.model";

@JsonObject
export class NivelReprezentareComisiiReportModel {

    @JsonProperty("rows", [NivelReprezentareComisiiReportRowModel])
    public rows: NivelReprezentareComisiiReportRowModel[] = [];

    @JsonProperty("totalPresedintiComisii", Number)
    public totalPresedintiComisii: number = null;

    @JsonProperty("totalVicepresedintiComisii", Number)
    public totalVicepresedintiComisii: number = null;

    @JsonProperty("totalReprezentareOrganismeInterne", Number)
    public totalReprezentareOrganismeInterne: number = null;

    @JsonProperty("totalReprezentareOrganismeInternationale", Number)
    public totalReprezentareOrganismeInternationale: number = null;

    @JsonProperty("sumaTotalReprezentare", Number)
    public sumaTotalReprezentare: number = null;

    @JsonProperty("totalProcentReprezentare", Number)
    public totalProcentReprezentare: number = null;
    
}

File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/report/NivelReprezentareComisiiReportModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.report;

import java.util.List;

public class NivelReprezentareComisiiReportModel {

	private List<NivelReprezentareComisiiReportRowModel> rows;
	private int totalPresedintiComisii;
	private int totalVicepresedintiComisii;
	private int totalReprezentareOrganismeInterne;
	private int totalReprezentareOrganismeInternationale;
	private int sumaTotalReprezentare;
	private double totalProcentReprezentare;
	
	public List<NivelReprezentareComisiiReportRowModel> getRows() {
		return rows;
	}
	public void setRows(List<NivelReprezentareComisiiReportRowModel> rows) {
		this.rows = rows;
	}
	public int getTotalPresedintiComisii() {
		return totalPresedintiComisii;
	}
	public void setTotalPresedintiComisii(int totalPresedintiComisii) {
		this.totalPresedintiComisii = totalPresedintiComisii;
	}
	public int getTotalVicepresedintiComisii() {
		return totalVicepresedintiComisii;
	}
	public void setTotalVicepresedintiComisii(int totalVicepresedintiComisii) {
		this.totalVicepresedintiComisii = totalVicepresedintiComisii;
	}
	public int getTotalReprezentareOrganismeInterne() {
		return totalReprezentareOrganismeInterne;
	}
	public void setTotalReprezentareOrganismeInterne(int totalReprezentareOrganismeInterne) {
		this.totalReprezentareOrganismeInterne = totalReprezentareOrganismeInterne;
	}
	public int getTotalReprezentareOrganismeInternationale() {
		return totalReprezentareOrganismeInternationale;
	}
	public void setTotalReprezentareOrganismeInternationale(int totalReprezentareOrganismeInternationale) {
		this.totalReprezentareOrganismeInternationale = totalReprezentareOrganismeInternationale;
	}
	public int getSumaTotalReprezentare() {
		return sumaTotalReprezentare;
	}
	public void setSumaTotalReprezentare(int sumaTotalReprezentare) {
		this.sumaTotalReprezentare = sumaTotalReprezentare;
	}
	public double getTotalProcentReprezentare() {
		return totalProcentReprezentare;
	}
	public void setTotalProcentReprezentare(double totalProcentReprezentare) {
		this.totalProcentReprezentare = totalProcentReprezentare;
	}
	
}
